src/sync.rs

```diff
@@ -665,7 +665,9 @@ const fn buffer_index(i: usize) -> usize {
     (((usize::BITS + 1 - (i + 1).leading_zeros()) >> 1) - 1) as usize
 }
 
-const NUM_BUFFERS: usize = (usize::BITS >> 2) as usize;
+/// Each buffer covers 2 bits of address space, so we need half as many
+/// buffers as bits in a usize.
+const NUM_BUFFERS: usize = (usize::BITS / 2) as usize;
 
 /// Append-only threadsafe version of `std::vec::Vec` where
 /// insertion does not require mutable access.
```

Comment:
the comment below about being 64 ptrs large is incorrect now

Reply:
It was wrong before, too 😅

Reply:
Fixed the comment below